doc/man3/BIO_ADDR.pod

```diff
@@ -39,6 +40,9 @@ BIO_ADDR_new() creates a new unfilled B<BIO_ADDR>, to be used
 with routines that will fill it with information, such as
 BIO_accept_ex().
 
+BIO_ADDR_copy() copies the contents of B<src> into B<dst>. Neither B<src> or
+B<dst> can be NULL.
+
```

Comment:
We use `I<src>` nowadays... I understand that changing it in the whole manpage is out of scope for the PR though.